wait/host_port.go

```diff
@@ -123,11 +126,25 @@ func (hp *HostPortStrategy) WaitUntilReady(ctx context.Context, target StrategyT
 		}
 	}
 
+	if err := externalCheck(ctx, ipAddress, port, target, waitInterval); err != nil {
+		return err
+	}
+
+	err = internalCheck(ctx, internalPort, target)
+	if err != nil && errors.Is(errShellNotExecutable, err) {
+		log.Printf("Shell not executable in container, only external port check will be performed")
```

Comment:
We probably want to add a `\n`
```suggestion
		log.Println("Shell not executable in container, only external port check will be performed")
```